src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
> we know problem exist in Chrome 34, but it might also be present in 33

That case is covered as well, assuming a fix immediately after version M: `Support: Chrome <=34 - M only`. If we don't know the fix version because that is _also_ too far back in the past, then I'd support either fudging it or extending the grammar to accept `Support: Chrome <=34 - <39 only`.